include/sharg/parser.hpp

```diff
@@ -591,13 +567,12 @@ class parser
      *            Super important integer for age.
      *```
      */
-    void
-    add_list_item(std::string const & key, std::string const & desc, option_spec const spec = option_spec::standard)
+    void add_list_item(std::string const & key, std::string const & desc, bool const advanced_only = false)
     {
         std::visit(
             [&](auto & f)
```

Comment:
```suggestion
            [&key, &desc, advanced_only](auto & f)
```

Mainly because of consistency :)